common-ktx/src/androidTest/java/co/nimblehq/common/extensions/SharedPreferencesExtTest.kt

```diff
@@ -0,0 +1,121 @@
+package co.nimblehq.common.extensions
+
+import android.content.Context
+import android.content.SharedPreferences
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.filters.SmallTest
+import androidx.test.platform.app.InstrumentationRegistry
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.Matchers.`is`
+import org.hamcrest.Matchers.isA
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+@SmallTest
```

Comment:
From my understanding, this test should not be SmallTest 🤔  Refer: https://testing.googleblog.com/2010/12/test-sizes.html

Reply:
@hoangnguyen92dn I will remove this kind 🙏